doc_src/add_doc_to_wheel.py

```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env python3
+
+"""
+Adds doc directory to exisiting wheel
+"""
+
+import subprocess
+import sys
+import os
+import tempfile
+import glob
+import shutil
+
+doc_zipfile = os.path.abspath(sys.argv[1])
+wheel_path = os.path.abspath(sys.argv[2])
+
+if not (doc_zipfile.endswith('.zip') and wheel_path.endswith('.whl')):
+    raise ValueError('Usage: add_doc_to_wheel docs.zip snappy.whl')
+
+python = sys.executable
+tmp_dir = tempfile.mkdtemp()
```

Comment:
with tempfile.TemporaryDirectory() as tmp_dir:

might be more pythonic. Not sure whether cleaning up the tmp directory at the end is a feature (making debugging easier) or bug.

Reply:
It was intended to make debugging easier.